reconcile/integrations_manager.py

```diff
@@ -1,39 +1,23 @@
-import json
+# import json
 import logging
 import os
 import sys
-from abc import (
-    ABC,
-    abstractmethod,
-)
-from collections.abc import (
-    Iterable,
-    Mapping,
-    MutableMapping,
-)
-from dataclasses import dataclass
-from typing import (
-    Any,
-    Optional,
-)
+
+# from dataclasses import dataclass
+from typing import Any, Optional, Iterable, Mapping, Sequence
```

Comment:
We agreed that collections should now be imported from `collections.abc` for [PEP-585](https://peps.python.org/pep-0585/) compliance.